helm/superset/values.yaml

```diff
@@ -267,20 +267,18 @@ supersetNode:
   # -- If true, forces deployment to reload on each upgrade
   forceReload: false
   # -- Init containers
-  # @default -- a container waiting for postgres to be up
+  # @default -- a container waiting for postgres
   initContainers:
     - name: wait-for-postgres
-      image: "{{ .Values.initImage.repository }}:{{ .Values.initImage.tag }}"
-      imagePullPolicy: "{{ .Values.initImage.pullPolicy }}"
+      image: jwilder/dockerize
```

Comment:
Shouldn't this be a reference to the config keys?

Reply:
Yeah, this is a good catch, and I probably should have mentioned it. I've hesitated on the best approach here.

The reason I'm not using the provided values, is to avoid risks of regressions, if, for some reason, someone has overridden these keys, the `dockerize` command will fail.

I don't think there's really any strong reason to be overriding these, but it's possible...

There might be a more elegant way to manage this potentially breaking change...

Reply:
I think it should be configurable in this case. the pull policy also :). We can just bump the chart version to indicate there's a potentially "breaking" change.

Reply:
It is already all configurable, as the entire `initContainers` section is provided in the `values.yaml`. The default can be overridden directly. It's just that it won't be using `initImage.repository`, but rather `supersetNode.initContainers[0].image`.

The version will definitely be bumped, as I think chart versions should be immutable. I believe the chart release process will ensure this anyway.

Reply:
To reduce a bit the redundancy, we can use `initImage.repository` as a default, but still leave the possibility to independently override it separately for each deployment...